docs/developers/Firmware/http-api.mdx

```diff
@@ -5,7 +5,7 @@ sidebar_label: HTTP API
 ---
 
 :::info
-This is a mini-spec on a HTTP API which can be used by browser based clients to do operations with devices.
+This is a mini-spec of the HTTP REST API which can be used by browser based clients to interact with Meshtastic devices.
```

Comment:
It's not a REST API, as the type of request is embedded in the protobuf payload and has no bearing on the http method.